horace_core/sqw/@d0d/d0d.m

```diff
@@ -29,5 +29,32 @@
         %            %
         %               obj = sqw(S);
         %        end
+        function obj = loadobj(S)
+            % Load a sqw object from a .mat file
+            %
+            %   >> obj = loadobj(S)
+            %
+            % Input:
+            % ------
+            %   S       An instance of this object or struct
+            %
+            % -------
+            % Output:
+            %   obj     An instance of this object
+            obj = d0d(S);
```

Comment:
I would move this after the `if` to keep it obvious that this has no effect on the if clause.